### DISCUSSION TITLE
Web radio won't play

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1682] [Fixed Issues]
After months of using rAudio to listen to BBC Radio 2 via a Minimserver stream (http://192.168.86.250:9790/minimstreamer/*/TESTR2/) due to the BBC tightening its access to streamed services, today this fails with 'Station not reachable'.  The same url WILL however play via VLC.  If I use the direct url that this Minimserver stream points (http://as-hls-uk-live.akamaized.net/pool_904/live/uk/bbc_radio_two/bbc_radio_two.isml/bbc_radio_two-audio=320000.norewind.m3u8) then rAudio will play the stream from this url.  The same is true of any web radio stations streamed via Minimserver (there have been no Minimserver updates recently). All will play as expected via VLC so I don't think this is a Minimserver issue.

I updated to 20240603 this morning - so what has changed to cause this to happen ?

Chris

**Comment by R101G** [comment]
@ceejayemm 
I have been using :

http://as-hls-ww-live.akamaized.net/pool_904/live/ww/bbc_radio_two/bbc_radio_two.isml/bbc_radio_two-audio=320000.norewind.m3u8

without problems ever since they shut down the mp3 streams. It's annoying not to have meta-data, but you should only be using the Sounds app, like you are told to.
The ww makes a difference. I don't know why.

**Reply by ceejayemm** [reply]
@R101G 

Thanks for your comments, you will not that the BBC url that works is basically the same as yours apart from uk/ww.  I can use 'uk' because I am based in UK, i believe 'ww' is only necessary if you are based outside the UK.

Ideally I would use the BBC Sounds app but I don't like choice being restricted for no good reason and the BBC have not yet explained clearly enough why they won't support an internet stream. Beside which I can't use the BBC Sounds app directly from within rAudio.  I would need to stream the app output from mu phone, using a data connection, via Bluetooth tho rAudio.  So although I could I don't particularly want to. I tried Daphile which does support the Sounds app but it that app was too complicated for me.

The issue here is something that worked yesterday isn't working today.  The only think that changed was an update to rAudio and there was nothing in the release 'notes' to say this was going to be a problem.

Regards.

Chris

**Reply by R101G** [reply]
@ceejayemm 

I'm not sure, but I think uk did not work for me, even though I am in the UK, and I believe these are the links which Alexa uses. I am using 20240603 with kernel 6.6.31-2-rpi aarch64.

I am pretty sure that the closure of the streams is about monetisation. The amount of audience data which iPlayer and Sounds collects is pretty impressive. I am sure they would shut down DAB and FM too, if they could get away with it. If I had to use the appalling Sounds app, I would not bother listening to the BBC.

**Reply by ceejayemm** [reply]
@R101G 
I have had no problems using the 'uk' url version for quite sometime and it still works for me today - but not via the Minimserver stream.  I am using a system based on i20240804 updated to 20240603 which gives me the 6.6.25-1-rpi aarch64 kernel on an RPi 3B+ - so very similar to you but I tend not to use pacman to update rAudio unless really necessary.

I totally agree with you regarding the way the BBC is going right now both technically and content wise.  I am tending to switch more and more to Greatest Hits Radio but I don't like the ads there (but you learn to live with them) and Bauer the owners of GHR are going down the same monetisation as the BBC.

Chris

**Comment by R101G** [comment]
@ceejayemm 

I am a bit confused. The stream I use in the rAudio library works, so I do not see how the Minimserver not working can be an rAudio problem?

**Reply by ceejayemm** [reply]
@R101G 
The direct url does work in rAudio.  The version I use via Minimserver does not work today with the message 'Station not reachable' BUT does work via VLC - which makes me think it is not a Minimserver issue.  The Minimserver version DID work yesterday BEFORE I applied the 20240603 update - this points me to this being an rAudio issue.

Chris

**Comment by kan-ibal** [comment]
Beg your pardon for OT but what is the reason for using Mimserver? This is additional, unnecessary proxy that transcodes a stream and worsening a quality of it. 
What is the type of output stream from Minimserver? Maybe it is a culprit?

**Comment by ceejayemm** [comment]
Seems to be fixed with 20240612. Thanks

Chris
